package.json

```diff
@@ -48,6 +48,7 @@
   "dependencies": {
     "@uswds/uswds": "3.1.0",
     "markdown-it-attrs": "^4.1.6",
+    "gulp": "^4.0.2",
```

Comment:
**comment** technically for best practices we SHOULD break up adding in new dependencies from PRs that add other features. The reasoning being, dependencies CAN break something so it is easier to track down errors if it isn't mixed with other changes. That being said, don't take the time to remove this, it is just an fyi (and one that I say knowing I still sometimes do this because I don't want to make yet another branch and PR)